pkg/runtime/types.go

```diff
@@ -38,8 +38,6 @@ type TypeMeta struct {
 
 const (
 	ContentTypeJSON string = "application/json"
-	// TODO: Fix the value.
-	ContentTypeProtobuf string = "application/protobuf"
```

Comment:
Why do you remove it? We should probably just change the value to:
application/vnd.kubernetes.protobuf